src/Site/views/shared/page/scriptoutput.html.twig

```diff
@@ -1,30 +0,0 @@
-{% extends app.website.base~'.html.twig' %}
```

Comment:
the concept I think is still good, however this file is so old that it's best to just get rid of it